CHANGES.md

```diff
@@ -74,6 +74,9 @@
 
 * MultiMap side inputs are now supported by the Go SDK ([BEAM-3293](https://issues.apache.org/jira/browse/BEAM-3293)).
 * Side inputs are supported within Splittable DoFns for Dataflow Runner V1 and Dataflow Runner V2. ([BEAM-12522](https://issues.apache.org/jira/browse/BEAM-12522)).
+* Upgrades Log4j version used in test suites to 2.16.0([BEAM-13434](https://issues.apache.org/jira/browse/BEAM-13434)).
```

Comment:
cc: @aaltay  FYI

Reply:
Could you change test suites to "test suites (Apache Beam testing environment only, not for end user consumption)"

Reply:
sure